docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -236,5 +241,12 @@ spec:
 If a service binding is declared using these label selectors to pick up workloads, SBO will
 periodically attempt to find and bind new workloads that match the given label selector.
 
-As a note: it is currently forbidden to attempt a binding with both `name:` and `labelSelector`
-defined (or `selector` in the spec API).  Any attempt to do so will result in a error.
+[Important]
+====
+Currently, it is forbidden to attempt a binding with both of the following fields defined: `name:` and `labelSelector`
+
+. `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
+. `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
+
+Any attempt to do so will result in a error.
+====
```

Comment:
```suggestion
[IMPORTANT]
====
Currently, it is forbidden to attempt a binding with the following fields defined: 

. `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
. `name:` and `selector` in the `servicebinding.io` API (Spec API)

Any attempt to do so will result in an error.
====
```
Request: Ensure to carefully consider my suggestions and apply them. I had given [this suggestion](https://github.com/redhat-developer/service-binding-operator/pull/1130/files#r872693629) previously as well. 

If this suggestion is technically incorrect and needs more clarification, please set up a meeting and we can resolve it soon together. We can have one meeting to cover both Workload resource mapping and Label selector in one go. Thanks!